its/projects/NuGet.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+      <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+  </packageSources>
+  <trustedSigners>
+    <repository name="nuget.org" serviceIndex="https://api.nuget.org/v3/index.json">
+        <certificate fingerprint="0e5f38f57dc1bcc806d8494f4f90fbcedd988b46760709cbeec6f4219aa6157d" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
+        <certificate fingerprint="5a2901d6ada3d18260b9c6dfe2133c95d74b9eef6ae0e5dc334c8454d1477df4" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
+        <!-- StyleCop.Analyzers: sharwell -->
+        <!-- NUnit: charliepoole;OsirisTerje;rprouse -->
+        <!-- coverlet.collector: tonerdo, dotnetfoundation, MarcoRossignoli, daveMueller -->
+        <!-- Microsoft.VisualStudio.UnitTesting: lizoc -->
+        <!-- NUnit.Analyzers: mikkelbu -->
+        <!-- xunit.core dotnetfoundation xunit -->
+        <owners>protobuf-packages;Microsoft;sharwell;dotnetfoundation;castleproject;jonorossi;onovotny;fluentassertions;jamesnk;joshclose;charliepoole;OsirisTerje;rprouse;tonerdo;dotnetfoundation;MarcoRossignoli;daveMueller;lizoc;mikkelbu;xunit</owners>
+      </repository>
+  </trustedSigners>
```

Comment:
From my understanding, you need to set `signatureValidationMode` to `require` when using `trustedSigners`:
```xml
<config>
  <clear />
  <add key="signatureValidationMode" value="require" />
</config>
```
As it is now, I am not sure it does the validation as the default value is `accept`.

[Source](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#config-section)

Reply:
Oh, it does. This is why we had the issue in the first place.
Anyway, I can add this to be specific.